core/src/main/res/values-b+be+tarask+old/strings.xml

```diff
@@ -1,8 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Authors:
```

Comment:
Please do not commit this file, it is automatically updated at runtime.

Reply:
I think these files are automatically updated when I have committed

Reply:
Yes, these tarask, and tarask+old files are automatically generated while the build task is ongoing. Please do not push these changes as these are automatically done via Gradle.

Reply:
Done sir